ton_sdk/src/local_tvm.rs

```diff
@@ -14,29 +14,38 @@
 
 use crate::error::*;
 use std::sync::Arc;
-use chrono::prelude::*;
-use std::convert::TryFrom;
 use ton_vm::executor::Engine;
 use ton_block::{
     Message,
     Serializable,
     Deserializable,
+    MsgAddressInt
 };
 use ton_block::error::*;
 use ton_vm::stack::{CellData, IntegerData, SaveList, SliceData, Stack, StackItem};
+use ton_types::HashmapE;
 use ton_vm::SmartContractInfo;
+use ton_vm::executor::gas::gas_state::Gas;
 
 #[cfg(test)]
 #[path = "tests/test_local_tvm.rs"]
 mod tests;
 
 #[allow(dead_code)]
-pub fn local_contract_call(code: Arc<CellData>, data: Option<Arc<CellData>>, msg: &Message)
--> SdkResult<Vec<Message>> {
+pub fn local_contract_call(
+    balance: u128,
+    balance_other: HashmapE,
+    address: &MsgAddressInt,
+    config_params: Option<Arc<CellData>>,
+    timestamp: u32,
+    code: Arc<CellData>,
+    data: Option<Arc<CellData>>,
+    msg: &Message)
+-> SdkResult<(Vec<Message>, i64)> {
```

Comment:
tuple again

Reply:
Ley it be tuple here. Made struct in wrapper functions